fury/ui.py

```diff
@@ -3504,14 +3505,174 @@ def __init__(self, labels, checked_labels, padding=1, font_size=18,
                                           checked_labels=checked_labels)
 
     def _handle_option_change(self, option):
-        for option_ in self.options:
+        for option_ in self.options.values():
             option_.deselect()
 
         option.select()
         self.checked_labels = [option.label]
         self.on_change(self)
 
 
+class ScrollBar(UI):
+
+    def __init__(self, ListBox2D):
```

Comment:
It can be a panel and `ListBox2D` is a panel improved, so replace `ListBox2D` by `panel`